src/tosam.py

```diff
@@ -8,9 +12,11 @@ def main():
         type=argparse.FileType('r')
     )
     args = argparser.parse_args()
-    print(args)
+    # print(args)
     for line in args.mas:
         chrom, read_name, read_str, pos = line.split('\t')
+
+        print(read_name.strip(), chrom.strip(), str(int(pos)+1), cigar = f"{len(read_str.strip())}M", read_str.strip(), sep = '\t')
```

Comment:
This is not what I meant by `cigar = ...`. That would be if you put the CIGAR string in a variable. Here you are trying a keyword argument, for a parameter `print` doesn't take, and before a positional which is always an error.

Reply:
I was a bit quick to paste it in... Now it's fixed! Thanks so much for the help :)

Reply:
No worries. It looks good now!